src/TestFramework/Coverage/JUnit/MemoizedTestFileDataProvider.php

```diff
@@ -33,11 +33,13 @@
 
 declare(strict_types=1);
 
-namespace Infection\TestFramework\Coverage\XmlReport;
+namespace Infection\TestFramework\Coverage\JUnit;
 
 use function array_key_exists;
 
 /**
+ * Caches parsed rows from JUnit report.
```

Comment:
Is that comment necessary? That is what memoizing is about

Reply:
See, I didn't know that when I wrote this comment. And it wasn't obvious it is related to JUnit report. Can we keep it?

Or, say, rename the class so it'd be more obvious what it does.